src/ontology/cl-edit.owl

```diff
@@ -32490,6 +32491,19 @@ AnnotationAssertion(rdfs:label obo:CL_4052004 "intestinal fibroblastic reticular
 SubClassOf(obo:CL_4052004 obo:CL_0009101)
 SubClassOf(obo:CL_4052004 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001962))
 
+# Class: obo:CL_4052006 (intestinal enteroendocrine progenitor)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31756561") Annotation(oboInOwl:hasDbXref "PMID:32531023") Annotation(oboInOwl:hasDbXref "PMID:35913117") Annotation(oboInOwl:hasDbXref "PMID:38260422") Annotation(oboInOwl:hasDbXref "doi:/10.1038/s41467-021-27901-5") obo:IAO_0000115 obo:CL_4052006 "An epithelial cell that is part of the crypt of Lieberkuhn, originating from intestinal stem cells and giving rise to enteroendocrine cells (EECs). In mouse and human, it expresses markers like Neurog3, and has the ability to proliferate and differentiate into various EEC subtypes. Its proliferative potential supports both crypt growth and cell proliferation, distinguishing it from fully differentiated EECs.")
```

Comment:
In the sentence:
> Its proliferative potential supports both crypt growth and cell proliferation, ...

What does it mean that the proliferative potential supports cell proliferation? I am not sure if it is its own proliferation or other cells, but in any case I would remove.